### PR TITLE
[AUTO BATCH PLUGIN] renaming the class name in auto batch plugin & remove namespace

### PR DESCRIPTION
**Pull request by zhaixuejun1993**
### Details:
 - using ov::... namespace
 - rename class, algin with template plugin naming style
 - rename class member name

### Tickets:
 - 111940

### CODE REVIEW

File: src/plugins/auto_batch/src/compiled_model.cpp [hunk 1]
```diff
@@ -9,90 +9,89 @@
...
+                             const std::set<std::string>& batchedOutputs)
     : InferenceEngine::ExecutableNetworkThreadSafeDefault(nullptr,
                                                           std::make_shared<InferenceEngine::ImmediateExecutor>()),
-      _network{networkWithBatch},
-      _networkWithoutBatch{networkWithoutBatch},
-      _config{config},
-      _batchedInputs(batchedInputs),
-      _batchedOutputs(batchedOutputs) {
+      m_network_with_batch{networkWithBatch},
+      m_network_without_batch{networkWithoutBatch},
```
Comment (ilya-lavrenov) [inline src/plugins/auto_batch/src/compiled_model.cpp:23]: network => model

File: src/plugins/auto_batch/src/compiled_model.hpp [hunk 2]
```diff
@@ -12,64 +12,72 @@
...
     };
 
-    explicit AutoBatchExecutableNetwork(
-        const InferenceEngine::SoExecutableNetworkInternal& networkForDevice,
-        const InferenceEngine::SoExecutableNetworkInternal& networkForDeviceWithoutBatch,
-        const DeviceInformation& networkDevices,
-        const std::unordered_map<std::string, InferenceEngine::Parameter>& config,
-        const std::set<std::string>& batchedIntputs,
-        const std::set<std::string>& batchedOutputs);
+    explicit CompiledModel(const InferenceEngine::SoExecutableNetworkInternal& networkForDevice,
```
Comment (ilya-lavrenov) [inline src/plugins/auto_batch/src/compiled_model.hpp:35]: explicit can be removed

File: src/plugins/auto_batch/src/sync_infer_request.hpp [hunk 3]
```diff
@@ -8,42 +8,53 @@
...
+    using Ptr = std::shared_ptr<SyncInferRequest>;
+    explicit SyncInferRequest(const InferenceEngine::InputsDataMap& networkInputs,
+                              const InferenceEngine::OutputsDataMap& networkOutputs,
+                              CompiledModel::WorkerInferRequest& workerRequestPtr,
+                              int batch_id,
+                              int num_batch,
+                              const std::set<std::string>& batchedIntputs,
+                              const std::set<std::string>& batchedOutputs);
+
+    explicit SyncInferRequest(const std::vector<std::shared_ptr<const ov::Node>>& inputs,
```
Comment (ilya-lavrenov) [inline src/plugins/auto_batch/src/sync_infer_request.hpp:25]: explicit can be removed

**Review by ilya-lavrenov — APPROVED** [review]

File: src/plugins/auto_batch/src/sync_infer_request.hpp (hunk 3 above)
Comment (zhaixuejun1993) [reply src/plugins/auto_batch/src/sync_infer_request.hpp:25]: done

**Review by wangleis — APPROVED** [review]

**Pull request merged**